Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -261,7 +261,10 @@ class TouchableOpacity extends React.Component<Props, State> {
 
   componentDidUpdate(prevProps: Props, prevState: State) {
     this.state.pressability.configure(this._createPressabilityConfig());
-    if (this.props.disabled !== prevProps.disabled) {
+    if (
+      this.props.disabled !== prevProps.disabled ||
+      JSON.stringify(prevProps.style) !== JSON.stringify(this.props.style)
```

Comment:
You could also just check if `opacity` is defined in style, and skip the change, without having to `stringify` for simple comparison

```suggestion
      !flattenStyle(this.props.style)?.opacity
```

Reply:
Or maybe even
```suggestion
      flattenStyle(prevProps.style)?.opacity !== flattenStyle(this.props.style)?.opacity !== undefined
```

Reply:
@Krisztiaan Thank you! I changed it. @cortinico You can also see the new changes and consider that for your discussion.